isbn/man-sru.php

```diff
@@ -169,6 +170,12 @@
         echo "</table>\n";
         echo "<hr/>\n";
         echo '<div>Bestand der UB Mannheim: E</div>';
+    } else if ($size > 50) {
```

Comment:
Maybe, we should just double this value. Otherwise two additional spaces or newlines may trigger a difference...